forecast_tools.R

```diff
@@ -3,12 +3,35 @@ library(lubridate)
 library(zoo)
 library(ggplot2)
 
+#Get all model aic values and calculate akaike weights
+compile_aic_weights = function(forecast_folder='./predictions'){
+  weight_filenames = list.files(forecast_folder, full.names = TRUE, recursive = TRUE)
+  weight_filenames = weight_filenames[grepl('aic_weights',weight_filenames)]
+  all_weights=data.frame()
+  
+  for(this_weight_file in weight_filenames){
```

Comment:
It should be possible to do this without the explicit loop, right? Something like this?

```
map(weight_filenames, ~try(read.csv(.x, na.strings = '', stringsAsFactors = FALSE))) %>% 
    bind_rows()
```

This should also be much faster than building up the data frame row-by-row. See the second circle of [The R Inferno](http://www.burns-stat.com/pages/Tutor/R_inferno.pdf) for more information than you ever wanted about growing objects element-by-element.